docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-2.1-applications-linux-containers-aks-kubernetes.md

```diff
@@ -0,0 +1,180 @@
+
+### Create the image in Release mode
+
+We'll now create the image in **Release** Mode (ready for production) changing to Release as shown here and running the application as did before.
+
+![Toolbar option in VS to build in release mode.](media/select-release-mode.png)
+
+**Figure 4-41**. Selecting Release Mode
+
+If you execute the `docker image` command, you'll see both images created one for `debug` and the other for `release` mode.
+
+### Create a new Tag for the Image
+
+Each container image needs to be tagged with the `loginServer` name of the registry. This tag is used for routing when pushing container images to an image registry.
+
+You can view the `loginServer` name from the Azure portal, taking the information from the Azure Container Registry
+
+![Browser view of the Azure container registry name, on the top right.](media/loginServer-name.png)
+
+**Figure 4-42**. View of the name of the Registry
+
+Or by running the command
+
+```console
+az acr list --resource-group MSSampleResourceGroup --query "[].{acrLoginServer:loginServer}" --output table
+```
+
+![Console output from the above command.](media/az-cli-loginServer-name.png)
+
+**Figure 4-43**. Get the name of the registry using PowerShell
+
+In both cases, you'll obtain the name, in our example `mssampleacr.azurecr.io`
+
+Now you can Tag the image, taking the latest image (Release image), with the command:
+
+```console
+docker tag mssampleaksapplication:latest mssampleacr.azurecr.io/mssampleaksapplication:v1
+```
+
+After running the `docker tag` command, list the images with the `docker images` command, you should see the image with the new tag.
+
+![Console output from the docker images command.](media/tagged-docker-images-list.png)
+
+**Figure 4-44**. View of tagged images
+
+### Push the image into the Azure ACR
+
+Now you can push the image into the Azure ACR, using the command:
+
+```console
+docker push mssampleacr.azurecr.io/mssampleaksapplication:v1
+```
+
+This command takes a while uploading the images but gives you feedback in the process.
+
+![Console output from the docker push command: shows a character based progress bar for each layer.](media/uploading-image-to-acr.png)
+
+**Figure 4-45**. Uploading the image to the ACR
+
+You can see below the result you should get when the process completes:
+
+![Console output from the docker push command, finished, showing all layers or nodes.](media/uploading-docker-images-complete.png)
+
+**Figure 4-46**. View of nodes
+
+The next step is to deploy your container into your AKS Kubernetes cluster, for that you need a file (**.yml deploy file**) that, in this case, contains:
+
+![Kubernetes deploy file content.](media/kubernetes-deploy-file.png)
```

Comment:
wouldn't be better to show this as code instead?

Reply:
yep, converted.